build/commands/scripts/updatePatches.js

```diff
@@ -12,6 +17,7 @@ const chromiumPathFilter = (s) => s.length > 0 &&
   !s.endsWith('new_tab_page_view.xml') &&
   !s.endsWith('channel_constants.xml') &&
   !s.includes('google_update_idl') &&
+  !s.includes('elevation_service_idl') &&
```

Comment:
what is this here for?

Reply:
Fixed in https://github.com/brave/brave-core/pull/20754/commits/8c2839474cc9e53e4f4bd02b86a959fd0519f421